canvas_modules/common-canvas/__tests__/common-properties/controls/radioset-test.js

```diff
@@ -32,6 +32,16 @@ describe("radio renders and works correctly with different enum types", () => {
 		wrapper.unmount();
 	});
 
+	it("radioset tooltip with string enum Gini & Entropy are displayed ", () => {
+		const control = {
```

Comment:
This will be hard to maintain.  Can you only search on a specific tooltip using the data-id from the control?

Reply:
For this test, i was having a hard time selecting the element via data-id, what I did instead is targeting the tooltips within div[data-id='properties-ci-radioString']"  Selector.
 
<img width="836" alt="image" src="https://user-images.githubusercontent.com/89609176/176099158-06e4d35d-e0e1-4a02-ac40-401665c0c06e.png">